Tests/RxSwift/retryWithBehaviorTests.swift

```diff
@@ -129,16 +124,16 @@ class RetryWithBehaviorTests: XCTestCase {
             next(440, 1),
             next(450, 2),
             error(460, RepeatTestErrors.fatalError)]
-        
-        // provide simple predicate that always return true
+
+        // provide simple predicate that always returns true
```

Comment:
As other comments were modified to capitalize first letter, shouldn't we modify this as well for sake of consistency.

Reply:
Yup, for sure - I just did it as I go, it wasn't a purpose of this PR :) 
We can keep cleaning as part of this PR or as a separate one TBH.